src/overlays/actors/ovl_En_Fr/z_en_fr.c

```diff
@@ -1058,7 +1058,7 @@ void EnFr_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec
 }
 
 void EnFr_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static u64* eyeTextures[] = { 0x060059A0, 0x06005BA0 };
+    static void* eyeTextures[] = { 0x060059A0, 0x06005BA0 };
```

Comment:
Could you add symbols for those too?

Reply:
Idk, if it's not decompiled then no.

Reply:
If it isn't decompiled then you could do `extern void* D_060059A0;`

Reply:
they are in hex all over the codebase. itll be up to whoever does this object to replace the symbols